apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx

```diff
@@ -221,6 +226,165 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
 
+    /**
+     * Retrieves scopes with the details from the API.
+     */
+    const getScopes = () => {
+
+        getOIDCScopesList()
```

Comment:
Use this https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/oidc-scopes/api/oidc-scopes.ts#L83 instead.

Refer: https://github.com/wso2/identity-apps/blob/f9dde022fc4b18a19a503e5d1ec223926daf1904/apps/console/src/features/oidc-scopes/pages/oidc-scopes.tsx#L84

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3566/commits/66c501e3bafdfeba484d242847e688d8d0ee2611